src/main/java/Event.java

```diff
@@ -0,0 +1,32 @@
+public class Event extends Task {
+    protected String at;
```

Comment:
The name "at" doesn't seem very comprehensive and I wouldn't be able to tell what it is used for unless I know what an Event object is meant for and how to create one from the CLI. There is some discrepancy comparing this with your Deadline "by" property as it is of type LocalDateTime in Deadline whereas it's just a String here.